packages/salesforcedx-vscode-core/src/telemetry/telemetryReporter.ts

```diff
@@ -108,8 +108,10 @@ export default class TelemetryReporter extends vscode.Disposable {
   private getCommonProperties(): { [key: string]: string } {
     const commonProperties = Object.create(null);
     commonProperties['common.os'] = os.platform();
-    commonProperties['common.platformversion'] = (os.release() || '')
-      .replace(/^(\d+)(\.\d+)?(\.\d+)?(.*)/, '$1$2$3');
+    commonProperties['common.platformversion'] = (os.release() || '').replace(
```

Comment:
We shouldn't be directly using or modifying this cause we're going to replace it soon with the new telemetry release module.